deployer/templates/fluentd.yaml

```diff
@@ -150,6 +150,12 @@ objects:
               value: ${JOURNAL_SOURCE}
             - name: "JOURNAL_READ_FROM_HEAD"
               value: ${JOURNAL_READ_FROM_HEAD}
+            - name: "USE_MUX"
```

Comment:
Are the changes in this file even necessary? Do we intend to set this for a 1.4 cluster?